yarn.lock

```diff
@@ -6222,7 +6222,7 @@ tiny-emitter@^2.0.0:
 
 tmp@^0.0.33:
   version "0.0.33"
-  resolved "https://registry.yarnpkg.com/tmp/-/tmp-0.0.33.tgz#6d34335889768d21b2bcda0aa277ced3b1bfadf9"
+  resolved "http://registry.npm.taobao.org/tmp/download/tmp-0.0.33.tgz#6d34335889768d21b2bcda0aa277ced3b1bfadf9"
```

Comment:
This change shouldn't be included here.

Reply:
Oh, forget to add `-g` when change the registry